utils/dataset_manifest/README.md

```diff
@@ -0,0 +1,71 @@
+## Simple command line to prepare dataset manifest file
+
+### Steps before use
+
+When used separately from Computer Vision Annotation Tool(CVAT), the required modules must be installed
+
+```bash
+python3 -m venv .env
+. .env/bin/activate
+pip install -U pip
+pip install -r requirements.txt
+```
+
+### Using
+
+```bash
+usage: python create.py [-h] --type {video,images} [--chunk_size CHUNK_SIZE] manifest_directory sources [sources ...]
+
+positional arguments:
+  manifest_directory    Directory where the manifest file will be saved
+  sources               Source paths
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --type {video,images}
+                        Type of datset data
+  --chunk_size CHUNK_SIZE
```

Comment:
@Marishka17 , When is it critical to have `--chunk_size` option? Why is it important for the manifest?

Reply:
It is needed for video data. Namely, to determine whether iteration over them will be smooth or not. If the keyframes are not enough for smooth iteration, the user will receive a notification and will be considered informed that when creating a task with this data and this manifest, the chunk timeout may differ.

Reply:
@Marishka17 , can we guess the chunk size always? I don't think that the user will know the right answer for chunk size when he/she generates the manifest file. Also usually chunk size should be in a small range for videos (4-30). Right? Does the limitation already answer on our questions?